NARFO_BE/NARFO_BE/Models/_Member.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NARFO_BE.Models
+{
+    public class _Member
+    {
+        public int Id { get; set; }
+        public string MEMNO { get; set; }
+        public string Password { get; set; }
+        public string Branch { get; set; }
+        public string SalesMan { get; set; }
+        public string Username { get; set; }
+
+        public string Title { get; set; }
+        public string Firstname { get; set; }
+        public string SURNAME { get; set; }
+        public string IDNo { get; set; }
+        [Column("Physical Address")]
+        public string Physical_Address { get; set; }
```

Comment:
You are using PascalCase then why would you suddenly use underscores? keep it PascalCase

Reply:
its saved in that way  in the db .**thank you for your views (:**

Reply:
@RetroDavis ^